samples/ar-view-node/src/main/java/io/github/sceneview/sample/arviewnode/nodes/ImageNodeHelper.kt

```diff
@@ -0,0 +1,59 @@
+package io.github.sceneview.sample.arviewnode.nodes
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.drawable.Drawable
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.engine.DiskCacheStrategy
+import com.bumptech.glide.load.resource.bitmap.Downsampler
+import com.bumptech.glide.request.RequestOptions
+import com.bumptech.glide.request.target.CustomTarget
+import com.bumptech.glide.request.transition.Transition
+import com.google.ar.core.Anchor
+import io.github.sceneview.ar.ARSceneView
+import io.github.sceneview.ar.node.AnchorNode
+import io.github.sceneview.node.ImageNode
+import io.github.sceneview.sample.arviewnode.nodes.events.ImageNodeEvent
+
+class ImageNodeHelper(
+    private val context: Context,
+    private val sceneView: ARSceneView,
+    val onEvent: (ImageNodeEvent) -> Unit
```

Comment:
Should we give a more descriptive name to the callback?

Reply:
On this side not. It would make sense to give a more descriptive name to the actual call function in MainActivity.

Reply:
I renamed the callback function